brillig_vm/src/opcodes.rs

```diff
@@ -4,10 +4,25 @@ use serde::{Deserialize, Serialize};
 
 pub type Label = usize;
 
+/// Lays out various ways we might interpret memory when passing to a foreign call external from Brillig.
```

Comment:
```suggestion
/// Lays out various ways an external foreign call's input and output data may be interpreted inside Brillig.
/// This data can either be an individual register value or memory.
```

Reply:
Just wanted to highlight more clearly what is meant to pass to a foreign call. Does that mean the foreign call inputs or outputs? The enum represents both inputs/outputs so wanted to highlight that. And then inside the enumeration we go over what happens in both the input and output case. Feel free to accept the above suggestion or rewrite how you like